node/core/core.go

```diff
@@ -10,7 +10,8 @@ import (
 	"github.com/celestiaorg/celestia-node/libs/fxutil"
 
 	"github.com/celestiaorg/celestia-node/core"
-	"github.com/celestiaorg/celestia-node/service/header"
+	"github.com/celestiaorg/celestia-node/header"
+	coreexchange "github.com/celestiaorg/celestia-node/header/core"
```

Comment:
```suggestion
	headercore "github.com/celestiaorg/celestia-node/header/core"
```

Reply:
Why? Because header/core pkg can have other core related stuff and not necessarily exchange.